engine/src/main/java/org/hibernate/validator/constraints/Mod11Check.java

```diff
@@ -122,6 +122,15 @@
 	ProcessingDirection processingDirection() default ProcessingDirection.RIGHT_TO_LEFT;
 
 	/**
+	 * @return an array of custom weights to be used in sum calculation of Mod11 algorithm.
+	 * If the size of provided weights array is less than needed ({@code array.length < endIndex - startIndex})
+	 * the weights will be used repeatedly. For example if constant weight is needed the weights array
+	 * can be initialized as {@code weights = { someConstant }}.
+	 * If weights array is left blank (which is a default value) a classical Mod11 weights are used.
```

Comment:
Thinking more about it, is this actually used for the new constraints? If not, I don't think we should add it.